src/App.tsx

```diff
@@ -10,31 +11,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
-
       <div data-cy="TabsComponent">
         <div className="tabs is-boxed">
           <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
+            <Tabs
+              tabs={tabs}
+              selectedTabId={selectedTab.id}
+              onTabSelected={setSelectedTab}
+            />
           </ul>
         </div>
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
+        <div
+          className="block"
+          data-cy="tab-content"
+        >
+          {selectedTab.content}
```

Comment:
`Tabs` should display it, not `App`
![image](https://user-images.githubusercontent.com/99331793/190090825-88d53363-6b2f-4520-8f4e-3e4ba0ed6fc6.png)